src/Microsoft.Health.Dicom.Core/Extensions/DicomTagExtensions.cs

```diff
@@ -22,7 +23,30 @@ public static class DicomTagExtensions
         public static string GetPath(this DicomTag dicomTag)
         {
             EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
-            return dicomTag.Group.ToString("X4") + dicomTag.Element.ToString("X4");
+            return dicomTag.Group.ToString("X4").ToUpperInvariant() + dicomTag.Element.ToString("X4").ToUpperInvariant();
+        }
+
+        /// <summary>
+        /// Get default VR for dicom tag.
+        /// </summary>
+        /// <param name="dicomTag">The dicm tag</param>
+        /// <returns>The default VR.</returns>
```

Comment:
nit: there should probably be some mention that the method returns `null` under certain circumstances, probably in the `<returns>`, but a `<remarks>` may also be appropriate.

Reply:
I would use `<see langword="null">` instead of the word "null"

---
In reply to: [564902240](https://github.com/microsoft/dicom-server/pull/518#discussion_r564902240) [](ancestors = 564902240)